diff --git a/node_modules/@vue/compiler-sfc/dist/compiler-sfc.cjs.js b/node_modules/@vue/compiler-sfc/dist/compiler-sfc.cjs.js
index 4a246ef..30596d9 100644
--- a/node_modules/@vue/compiler-sfc/dist/compiler-sfc.cjs.js
+++ b/node_modules/@vue/compiler-sfc/dist/compiler-sfc.cjs.js
@@ -657,11 +657,8 @@ var scopedPlugin = postcss.plugin('vue-scoped', (options) => (root) => {
                     selector.insertAfter(
                     // If node is null it means we need to inject [id] at the start
                     // insertAfter can handle `null` here
-                    node, selectorParser.attribute({
-                        attribute: idToAdd,
-                        value: idToAdd,
-                        raws: {},
-                        quoteMark: `"`
+                    node, selectorParser.className({
+                        value: idToAdd
                     }));
                 }
             }
